public/documents/fari-games/charge-srd.md

```diff
@@ -7,19 +7,19 @@ highlightFont: Oswald
 
 # Introductions | Charge SRD
 
-This is a condensed version of the rules of Charge RPG that you can use to make your own **Powered by Charge** games.
+The Charge SRD is a condensed version of the rules of Charge RPG that you can use to make your own **Powered by Charge** games.
 
-This document follows a certain format that you can use as a base framework for your game.
+It follows a certain format that you can use as a base framework for your game, but also offers design advice so that you can hack it while still keeping the "Charge feeling"
 
 You can fill-in the gaps, or tear-it down to make something completely different.
 
 We can't wait to see what you will build with this. Enjoy!
 
 > #### Designer Note
 >
-> Text inside blockquotes is for the game designer.
+> Text inside blockquotes is for you, the game designer.
```

Comment:
Blockquote is kinda a markdown-centric term, is it worth clarifying "this is a blockquote" for people reading SRD?

```suggestion
> Text inside blockquotes (like this) is for you, the game designer.
```